.github/workflows/continous-integration.yml

```diff
@@ -78,7 +86,7 @@ jobs:
       # makes sure we catch any dependency error early. they will create strange
       # errors during the docs build, so easier to catch them early on by
       # trying to run the `rasa` command once before the docs build.
-      run: rasa --help
+      run: poetry run rasa --help
```

Comment:
why does this not work anymore? Because the virtualenv is not activated?

Reply:
yes, it's to make sure we run it in a virtualenv